index.js

```diff
@@ -263,13 +269,37 @@ module.exports = {
               __frameId: params.frameId,
               _initialPriority: request.initialPriority,
               _priority: request.initialPriority,
-              _initiator: params.initiator.url,
-              _initiator_line: params.initiator.lineNumber,
               pageref: currentPageId,
               request: req,
               time: 0
             };
 
+            switch (params.initiator.type) {
+              case 'script':
```

Comment:
According to the [tip-of-tree docs](https://chromedevtools.github.io/devtools-protocol/tot/Network/) for Chrome DevTools Protocol, even script sometimes has lineNumber and url. Maybe better to always set them if they're present. Then the switch can be replaced with a single ```if (params.initiator.type === 'script')``` to add line/url if missing + extra info from stack, such as function name.